src/main/java/com/limechain/runtime/hostapi/HostApi.java

```diff
@@ -50,69 +53,77 @@ protected static ImportObject getImportObject(final String functionName,
         }, args, EMPTY_LIST_OF_TYPES);
     }
 
+    @Deprecated(forRemoval = true)
     public static byte[] getDataFromMemory(long pointer) {
-        int ptr = (int) pointer;
-        int ptrLength = (int) (pointer >> 32);
-        byte[] data = new byte[ptrLength];
-
-        Memory memory = getMemory();
-        memory.buffer().get(ptr, data, 0, ptrLength);
-        return data;
+        return null;
```

Comment:
Maybe until removal provide a fallback implementation to the deprecated method
```suggestion
        return getDataFromMemory(new RuntimePointerSize(pointer));
```